wiki.moinmoin

```diff
@@ -187,9 +187,14 @@ RULE:   If any are present, this requires a more in-depth security review.
 TODO: - no `suid` or `sgid` binaries
 TODO-A: - no executables in `/sbin` and `/usr/sbin`
 TODO-B: - Binary <TBD> in sbin is no problem because <TBD>
-TODO: - Package does not install services
-TODO: - Package installs services, but they are safe because <TBD> (reduced
-TODO:   permissions, tmp env, restricted user, ...)
+TODO-A: - Package does not install services, timers or reoccuruing jobs
+TODO-B: - Package does install services, timers or reoccuruing jobs
+TODO-B:   <TBD> (list services, timers, jobs)
+TODO-C: - Package does install services, timers or reoccuruing jobs
+TODO-C:   <TBD> (list services, timers, jobs)
+TODO-C:   Those should be considered rather safe because <TBD> (add details like
+TODO-C:   reduced permissions, tmp env, restricted users/groups, seccomp,
+TODO-C:   apparmor, ... the more is added => more likely MIR team agrees)
```

Comment:
I would say "Package does install services, timers or recurring jobs (list) with the following security features:" then your list

Reply:
Yeah I like that, it is even further improving my first try on making it clear that the `decision` isn't up to the reporter.

Reply:
I combined the sentence with my section, pushing the new version.